src/System.Management.Automation/resources/ParserStrings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
Please avoid combining reformatting and functional fixes in the same change.  Suggestion:  Submit a PR for the reformatting before the functional change.